epsagon/utils.py

```diff
@@ -95,6 +95,24 @@ def get_tc_url(use_ssl):
     return TRACE_COLLECTOR_URL.format(protocol=protocol, region=REGION)
 
 
+def get_trace_log_config():
+    """Returns the trace log correlation configuration"""
+    # Default is True
+    logging_tracing_enabled = True
+
+    # If EPSAGON_LOGGING_TRACING_ENABLED exists as an env var - use it
+    if os.getenv('EPSAGON_LOGGING_TRACING_ENABLED'):
+        logging_tracing_enabled = (
+            os.getenv('EPSAGON_LOGGING_TRACING_ENABLED') or ''
```

Comment:
I think the `or ''` isn't required because you already checked for a value in EPSAGON_LOGGING_TRACING_ENABLED inside the if statement